apps/synapse-portal-framework/tsconfig.json

```diff
@@ -1,17 +1,39 @@
 {
   "extends": "../../shared/tsconfig.base.json",
   "compilerOptions": {
+    "composite": true,
```

Comment:
With TypeScript project references, projects that can be referenced by other projects must set `"composite": true`. This enforces a few other rules that can be seen in other changes in the tsconfig files.